docs/internals/orientation-vector.md

```diff
@@ -3,41 +3,39 @@ title: "Orientation Vector"
 linkTitle: "Orientation Vector"
 weight: 10
 type: "docs"
-description: "A description of Viam's orientation vector method."
+description: "A description of Viam's orientation vector object."
 ---
-Viam's orientation vector is a method for describing the orientation of an object in 3D space.
-It is part of a Pose which also includes the position in 3D space.
 
-## Basics
+An _orientation vector_ specifies the angle at which an object is oriented in 3D space.
+You use orientation vectors to specify relative orientations of components when using the [Motion Service](../../services/motion/) and [Frame System](../../services/frame-system/).
+The first three components of this vector form an axis pointing in the same direction as the object.
+**Theta** specifies the angle of the object's rotation about that axis.
 
-Orientation vectors are composted of 4 attributes.
+## Example: A camera in a room
 
-- RX, RY, RZ - define a vector from the center of the object to another point in the reference frame.
-This defines the direction something is pointing in.
-- Theta - describes the angular position around that vector.
+Imagine you have a room with a camera.
+The corner of the room is (0, 0, 0).
 
-## Examples
+To configure the camera into the frame system, you need to know where in the room the camera is, and where it's pointing.
 
-### Configuring frame for a camera
+![A camera in a 3D space](../img/vector/orientation-vector-camera.png)
 
-- You have a room with a camera.
-- The corner of the room is (0,0,0).
-- You want to configure the camera into the frame system, so you need to know where in the room the camera is, and where it's pointing.
-- This requires a Pose.
-- X, Y, Z are simply the measurements from the corner of the room.
-- Now to figure out the orientation vector:
-  - To figure out RX, RY, RZ first take a picture with the camera.
-  - Determine the point in the very center of the image.
-  - Measure the X, Y, Z of that point from the corner (call them X2, Y2, Z2, respectively).
-  - RX, RY, RZ become the difference between X, Y, Z (the camera's position) and X2, Y2, Z2 (that point's position).
-  - Theta is determined by looking at the picture and changing Theta's value until down is correct, likely 0, 90 or 180.
-    - To do this, take a picture
-    - Determine if the orientation is correct
-    - If not, add 90 to Theta, and try again
-![camera example](../img/vector/orientation-vector-camera.png)
+**(RX, RY, RZ)** are defined by measurements starting from the corner of the room to the camera:
 
-## Why we like it
+1. Determine the starting point.
+   In this case it is (0, 0, 0).
+2. Determine the position of the camera.
+   In this case it is (3, 5, 2).
+3. Subtract the starting point from the position of the camera: (3, 5, 2) - (0, 0, 0) = (3, 5, 2).
+
+**Theta** describes the angle of the camera.
```

Comment:
"angle of rotation of the camera around the vector calculated above."

Maybe mention you can think of theta as "roll" in a pitch-roll-yaw system?